package.json

```diff
@@ -4,35 +4,23 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "dev": "nodemon --exec babel-node src/index.js",
-    "start": "node dist/index.js",
     "build": "babel src --out-dir dist",
+    "start": "node dist/index.js",
+    "start:dev": "cross-env DEBUG=dev nodemon --exec babel-node src/index.js",
     "sync-env": "sync-dotenv",
-    "test": "nyc --reporter=text mocha ./tests/ --recursive -r @babel/register",
+    "test": "nyc --reporter=text mocha ./src/tests/ --recursive -r @babel/register",
```

Comment:
There is no directory name "tests". Can you please restore the "tests" directory and ensure your tests are passing before raising a PR?